auslib/web/admin/views/releases.py

```diff
@@ -622,3 +622,48 @@ def get(self, change_id, field):
         )
 
         return Response('\n'.join(result), content_type='text/plain')
+
+
+class ScheduledReleaseFieldView(AdminView):
+
+    def __init__(self):
+        self.table = dbo.releases.scheduled_changes
+
+    def get_value(self, sc_id, field=None):
+        data = self.table.select(where={"sc_id": sc_id}, transaction=None)[0]
+        if not data:
+            abort(400, 'Bad sc_id')
+        if not field:
+            return data
+        if field not in data:
+            raise KeyError('Bad field')
+        return data[field]
+
+
+class ScheduledReleaseDiffView(ScheduledReleaseFieldView):
+    """/diff/:sc_id"""
+
+    def get_release(self, sc):
+        data = dbo.releases.select(where={"name": sc["base_name"], "product": sc["base_product"]}, limit=1)[0]
+        if not data:
+            abort(400, 'Bad sc_id')
+        return data
+
+    def get(self, sc_id):
+        sc = self.get_value(sc_id)
+        release = self.get_release(sc)
+
+        if 'data' not in release:
+            return problem(400, "Bad Request", "Bad field")
+
+        previous = json.dumps(release['data'], indent=2, sort_keys=True)
+        value = json.dumps(sc["base_{}".format('data')], indent=2, sort_keys=True)
+        result = difflib.unified_diff(
+            previous.splitlines(),
+            value.splitlines(),
+            fromfile="Data Version {}".format(release["data_version"]),
+            tofile="Data Version {}".format(sc["data_version"]),
```

Comment:
These labels aren't quite right - sc_data_version and data_version aren't really comparable in this way (the former tracks changes to the release, the latter tracks changes to a specific scheduled update to a release). Better labels would be "Current Version (Data Version X)" for fromfile, and "Scheduled Update" for tofile. The sc data_version isn't important to show here.